src/main/kotlin/com/matthewrussell/trchunkbrowser/domain/ObservableListExtensions.kt

```diff
@@ -0,0 +1,22 @@
+package com.matthewrussell.trchunkbrowser.domain
+
+import com.matthewrussell.trchunkbrowser.model.AudioSegment
+import javafx.collections.ObservableList
+import java.util.regex.Pattern
+
+fun ObservableList<AudioSegment>.sortedByLabel(): ObservableList<AudioSegment> {
```

Comment:
This would be better implemented as a function, not an extension method. Extension methods aren't really intended to replace functions, but provide extra utilities to core types, that would be useful to everyone and not specific to your own app.

For example, a capitalizeFirstLetter() extension method on the String class would be useful, probably should be in the string class anyway even though it isn't, and would be useful to basically everyone who uses String.

This language feature is also a little frustrating. It adds this method to all ObservableList objects in the package; even if they don't need it. It also isn't available outside just this package, and as a result could cause confusion as to why it doesn't show up somewhere else (or wondering where it came from if you see it and don't know why).

This is very specific, especially since it's not an extension method on ObservableList<T>